javascript/javascript.js

```diff
@@ -0,0 +1,74 @@
+function murphyInfo() {

```

Comment:
For all of these you will want to use one function, then take in information passed in from the HTML re-using the function for each click.